base/qserv.yaml

```diff
@@ -15,4 +15,4 @@ spec:
       dbimage: "mariadb:10.2.16"
   xrootd:
     image: "qserv/qserv:dcbfff7"
-    replicas: 4
+    replicas: 2
```

Comment:
it's kind of confusing when values get defined in `base` yaml and when they are always(?) overridden in `overlay` yaml. It gets really complicated in my head with image tags. Is there any way to indicate values that are expected to be overridden in an overlay, as opposed to values where the default/base value is expected to be reasonable & used? even some convention like `-` would be at least more clear (I don't know if this would lead to any kind of usable error reporting if it wasn't properly overridden tho?)
```xrootd:
    image: -```
But: is there a convention? What could be done to make this better?

Reply:
My mistake. Thanks for reporting this, I have cleanup this useless parameters from overlay files.